docs/configuration/identity-providers/oidc.md

```diff
@@ -206,23 +212,25 @@ required: no
 The maximum lifetime of an ID token. For more information read these docs about [token lifespan].
 
 ### refresh_token_lifespan
+
 <div markdown="1">
 type: string
-{: .label .label-config .label-purple } 
+{: .label .label-config .label-purple }
 default: 30d
 {: .label .label-config .label-blue }
 required: no
 {: .label .label-config .label-green }
 </div>
 
-The maximum lifetime of a refresh token. This should typically be slightly more the other token lifespans. This is 
-because the refresh token can be used to obtain new refresh tokens as well as access tokens or id tokens with an 
+The maximum lifetime of a refresh token. This should typically be slightly more the other token lifespans. This is
```

Comment:
```suggestion
The maximum lifetime of a refresh token. The
```
Since the lifespan can also be less than other tokens, I feel the sentence does not bring a lot of value. I suggest we should rather not influence people but let the administrators decide based on factual arguments and/or security requirements.